pkg/justification/processor.go

```diff
@@ -189,11 +194,20 @@ func (p *Processor) runValidations(ctx context.Context, req *jvspb.CreateJustifi
 		audiencesLength += len(v)
 	}
 	if got, max := audiencesLength, 1_000; got > max {
-		merr = errors.Join(merr, fmt.Errorf("audiences size (%d bytes) must be less than %d bytes",
+		validationErr = errors.Join(validationErr, fmt.Errorf("audiences size (%d bytes) must be less than %d bytes",
 			got, max))
 	}
 
-	return
+	if internalErr != nil {
+		logger.ErrorContext(ctx, "internal error during validation", internalErr)
+		return status.Errorf(codes.Internal, "unable to validate request")
+	}
+
+	if validationErr != nil {
+		logger.WarnContext(ctx, "failed to validate token", "error", validationErr)
+		return status.Errorf(codes.InvalidArgument, "failed to validate request: %s", validationErr.Error())
+	}
```

Comment:
Wonder how we can combine the errors...

Reply:
Errors from validation response are joined to `validationErr`, other errors thrown in this function are joined to  `internalErr`.  
For internal errors, we log them with `error` severity and only returns a standard internal error message to the user. For validation errors, we log them with `warning` severity and return the error messages to the user

Reply:
What if there are a mix of internalErr and validationErr?

Reply:
Yeah so the plan in https://github.com/abcxyz/jvs-plugin-jira/issues/48 is that we show a standard error for internal error to hide the details, and full error messages when validation fails. When internal error occurs, the validation won't succeed regardless of validation error, so I only return internal error to notify the user that server is not working properly.  The user will retry and see the validation error when server is back.

Reply:
Ok, sgtm, in this case, we can return early when internal error occurs. Since we might have a list of Justifications and a list of validators to call, I am also thinking if it makes sense to combine the errors (internal errors no details + validation errors with details), it will be strange since we can only set one status code though.

Reply:
I see your points. In the latest commit I included the validation error in logs as well when any internal error occur. Note that all validators are still invoked even when any one of them throws error